itests/org.jboss.tools.cdk.ui.bot.test/META-INF/MANIFEST.MF

```diff
@@ -5,7 +5,7 @@ Bundle-SymbolicName: org.jboss.tools.cdk.ui.bot.test
 Bundle-Version: 3.11.0.qualifier
 Bundle-Vendor: Red Hat
 Require-Bundle: org.junit,
- org.eclipse.reddeer.go;bundle-version="[2.1.0,3.0.0)",
+ org.eclipse.reddeer.go,

```

Comment:
is the reason for eliminating the upper bound that we'll be using or are already using red deer 3.0? In general I'd advocate for an upper bound that excludes versions where we'd expect an API breakage. If we're using 3.0 already we maybe should now use 4.0 as an upper bound?